### ISSUE TITLE
Multithreading for tagging

### ISSUE THREAD

**Issue by WilliamTambellini** [andre-martins/TurboParser#34]
Hi André
Looks like the tagger is not threadsafe : when tagging from different threads in parallel, some random crashes happen.
Could you summarize what would be needed in order for the tagger to be threadsafe ?
Cheers

**Comment by davidalbertonogueira** [comment]
Hi William, 
Are you using the libturboparser\TurboParserInterface?

**Comment by WilliamTambellini** [comment]
Hi David, Yes but sparsely because of this bad thread safety limitation.
